### ISSUE TITLE
No longer builds for stable version of iOS

### ISSUE THREAD

**Issue by deanishe** [rasmuslos/audiobooks#1]
Minimum iOS version is now 17, which hasn't been released yet. Same for the version of Xcode required to build.

Could you add a branch or tag that points to a version that doesn't require beta software to build and run?

Looking forward to trying the new features when iOS 17 comes out.

Keep up the great work.

**Comment by rasmuslos** [comment]
First of all, sorry for the late reply. I created the tag v1.0.0 for the latest iOS 16 version, as well as v1.1.0 for iOS 17. I hope this helps. But I don't think I will publish any major new features for this version of the app, as I have recently developed another iOS app that primarily focuses on audio, and learned a lot. I was not really happy with how some things turned out with this app in the first place, so I am planning on rewriting it using newer Technologies like SwiftData. Still nice to hear that you are enjoying using it :)
